src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -57,11 +58,17 @@ export default class DetailsBody extends React.Component<Props> {
         .filter(({slug}) => incident.projects.includes(slug))
         .map(({id}) => Number(id)),
       version: 2 as const,
-      range: `${incident.alertRule.timeWindow}m`,
+      start: incident.dateStarted,
+      end: incident.dateClosed ?? getUtcDateString(new Date()),
     };
 
     const discoverView = EventView.fromSavedQuery(discoverQuery);
-    return discoverView.getResultsViewUrlTarget(orgId);
+    const {query, ...toObject} = discoverView.getResultsViewUrlTarget(orgId);
+
+    return {
+      query: {...query, interval: `${incident.alertRule.timeWindow}m`},
```

Comment:
@billyvg I don't think this `interval` query string is actually being passed anywhere. For example hardcode this to `3m`. None of the API endpoint calls is making use of this.

Reply:
It needs this https://github.com/getsentry/sentry/pull/17119

Reply:
Ah gotcha 👍